jedi/plugins/stdlib.py

```diff
@@ -781,6 +781,63 @@ def _os_path_join(args_set, callback):
 }
 
 
+def infer_django_field(cls, field):
+    field_tree_instance, = field.infer()
+
+    if field_tree_instance.name.string_name in ('CharField', 'TextField', 'EmailField'):
+        model_instance_field_type, = cls.inference_state.builtins_module.py__getattribute__('str')
+        return DjangoModelField(model_instance_field_type, field).name
+
+    integer_field_classes = ('IntegerField', 'BigIntegerField', 'PositiveIntegerField', 'SmallIntegerField')
+    if field_tree_instance.name.string_name in integer_field_classes:
+        model_instance_field_type, = cls.inference_state.builtins_module.py__getattribute__('int')
+        return DjangoModelField(model_instance_field_type, field).name
+
+    if field_tree_instance.name.string_name == 'FloatField':
+        model_instance_field_type, = cls.inference_state.builtins_module.py__getattribute__('float')
+        return DjangoModelField(model_instance_field_type, field).name
+
+    if field_tree_instance.name.string_name == 'BinaryField':
+        model_instance_field_type, = cls.inference_state.builtins_module.py__getattribute__('bytes')
+        return DjangoModelField(model_instance_field_type, field).name
+
+    if field_tree_instance.name.string_name == 'BooleanField':
+        model_instance_field_type, = cls.inference_state.builtins_module.py__getattribute__('bool')
+        return DjangoModelField(model_instance_field_type, field).name
+
+    if field_tree_instance.name.string_name == 'DecimalField':
+        model_instance_field_type, = cls.inference_state.import_module(('decimal',)).py__getattribute__('Decimal')
+        return DjangoModelField(model_instance_field_type, field).name
+
+    if field_tree_instance.name.string_name == 'ForeignKey':
+        # TODO: infer related object class and make a filter for that class
+        return
```

Comment:
This could look something like

```
if isinstance(field_tree_instance, TreeInstance):
    # Private access, don't worry about it and let it be my problem
    argument_iterator = field_tree_instance._arguments.unpack()
    next(argument_iterator, None)
    key, lazy_values = next(argument_iterator, (None, None))
    if key is None and lazy_values is not None:
        for value in lazy_values.infer():
            string = value.get_safe_value(default=None)
            if isinstance(value, str):
                for v in cls.parent_context().py__getattribute__(value):
                    yield DjangoModelField(v, field)
            else:
                yield DjangoModelField(value, field)
```